ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -456,122 +539,88 @@ forkBlockProduction IS{..} =
   New network layer
 -------------------------------------------------------------------------------}
 
-data NetworkRequires m up blk = NetworkRequires {
```

Comment:
I liked the `NetworkRequires` record, it contained the server and clients of the protocol. Is there a reason for getting rid of it? Looking at `consensusNetworkApps`, I can't seem to find one.

Reply:
I renamed it as `NetworkApplication` (this is the network requirement), this matches better with names in `ouroboros-network`.